src/main/webapp/resources/js/components/menu/EllipsisMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Dropdown, Menu } from "antd";
```

Comment:
Sorry missed this, `Menu` is an unused import

Reply:
Removed in 288f7a6dcd3e02b97ceb3a5535ab6bb1fd0d5e70